modules/diskq/qdisk.c

```diff
@@ -1021,37 +1021,37 @@ _save_queue(QDisk *self, GQueue *q, QDiskQueuePosition *q_pos)
 gboolean
 qdisk_save_state(QDisk *self, GQueue *qout, GQueue *qbacklog, GQueue *qoverflow)
 {
+  if (self->options->reliable)
+    {
+      msg_info("Reliable disk-buffer state saved",
+               evt_tag_str("filename", self->filename),
+               evt_tag_long("qdisk_length", self->hdr->length));
+      return TRUE;
```

Comment:
Zeroing out `q*_pos` is probably a useful operation here, because changing disk-buffer type is somewhat possible.

Reply:
Nice catch, removed that commit.